postgres/README.md

```diff
@@ -20,28 +20,35 @@ The command deploys PostgreSQL on the Kubernetes cluster in the default configur
+| ---                                       | ---                                                      | ---                                         |
+| `secret.enabled`                          | Use secret to set postgres password                      | `false`                                     |
+| `secret.secretName`                       | Secret store name                                        | `nil`                                       |
+| `secret.postgresPassword`                 | Secret store key for postgres password                   | `nil`                                       |
+| `persistence.enabled`                     | Use a PVC to persist data                                | `false`                                     |
+| `persistence.storageClass`                | Storage class of backing PVC                             | `nil` (uses alpha storage class annotation) |
+| `persistence.accessMode`                  | Use volume as ReadOnly or ReadWriteOnce                  | `ReadWriteOnce`                             |
+| `persistence.annotations`                 | Persistent Volume annotations                            | `{}`                                        |
+| `persistence.size`                        | Size of data volume                                      | `8Gi`                                       |
+| `persistence.subPath`                     | Subdirectory of the volume to mount at                   | `postgresql-db`                             |
+| `resources`                               | CPU/Memory resource requests/limits                      | Memory: `256Mi`, CPU: `100m`                |
+| `service.port`                            | TCP port                                                 | `5432`                                      |
+| `service.type`                            | k8s service type exposing ports, e.g. `NodePort`         | `ClusterIP`                                 |
+| `initData.enabled`                        | Initialize volume with data from given source            | `false`                                     |
+| `initData.datafile`                       | Name of the archive to download                          | `subsets.bin.tgz`                           |
+| `initData.objectStorage.provider`         | Povider of the source, e.g. minio or s3                  | `minio`                                     |
+| `initData.objectStorage.bucket`           | Bucket name                                              | `subsets`                                   |
+| `initData.objectStorage.image`            | Image to use in the init container                       | `minio/mc:latest`                           |
+| `initData.objectStorage.host`             | Host of the object storage                               | `https://minio.k8s.jobteaser.net`           |
```

Comment:
This value has no default.

Reply:
The default value should not be `jobteaser` configuration. I think is more clear if we don't use default value for the init data.